### PR TITLE
Add JAXB to dependencies to be able to compile with java 9 and newer

### PR DESCRIPTION
**Pull request by GitAntoinee**
I added JAXB to dependencies to be able to compile with java 9 and newer (I tried with `openjdk 14.0.1 2020-04-14`).

### CODE REVIEW

**Comment by Janmm14** [comment]
you probably meant to be able to compile with java 9 and **newer**

**Comment by GitAntoinee** [comment]
You're right, I do not speak English very well unfortunately

File: src/main/java/com/javadeobfuscator/deobfuscator/transformers/allatori/StringEncryptionTransformer.java [hunk 1]
```diff
@@ -167,12 +167,12 @@ public boolean canCheckEquality(JavaValue first, JavaValue second, Context conte
     private boolean isAllatoriMethod(Map<Integer, AtomicInteger> insnCount, Map<String, AtomicInteger> invokeCount) {
         //XXX: Better detector
     	if(insnCount.get(Opcodes.IXOR) == null ||
-    		insnCount.get(Opcodes.ISHL) == null ||
+    		//insnCount.get(Opcodes.ISHL) == null ||
```
Comment (Janmm14) [inline src/main/java/com/javadeobfuscator/deobfuscator/transformers/allatori/StringEncryptionTransformer.java:170]: unrelated change, should be left out

File: src/main/java/com/javadeobfuscator/deobfuscator/transformers/allatori/StringEncryptionTransformer.java [hunk 2]
```diff
@@ -167,12 +167,12 @@ public boolean canCheckEquality(JavaValue first, JavaValue second, Context conte
...
     		insnCount.get(Opcodes.NEWARRAY) == null ||
     		invokeCount.get("charAt") == null || invokeCount.get("length") == null)
     			return false;
         return insnCount.get(Opcodes.IXOR).get() >= 3 &&
-               insnCount.get(Opcodes.ISHL).get() >= 1 &&
+               //insnCount.get(Opcodes.ISHL).get() >= 1 &&
```
Comment (Janmm14) [inline src/main/java/com/javadeobfuscator/deobfuscator/transformers/allatori/StringEncryptionTransformer.java:175]: unrelated change, should be left out

File: src/main/java/com/javadeobfuscator/deobfuscator/executor/defined/JVMMethodProvider.java [hunk 3]
```diff
@@ -359,6 +359,15 @@
...
         }});
+        put("java/lang/LinkageError", new HashMap<String, Function3<JavaValue, List<JavaValue>, Context, Object>>() {{
+            put("<init>()V", (targetObject, args, context) -> {
+                expect(targetObject, "java/lang/LinkageError");
+                targetObject.initialize(null);
+                return null;
+            });
+            put("getStackTrace()[Ljava/lang/StackTraceElement;", (targetObject, args, context) -> context.getStackTrace());
+            put("toString()Ljava/lang/String;", (targetObject, args, context) -> targetObject.toString());
+        }});
```
Comment (Janmm14) [inline src/main/java/com/javadeobfuscator/deobfuscator/executor/defined/JVMMethodProvider.java:370]: unrelated change, should be left out

**Review by Janmm14 — CHANGES_REQUESTED** [review]

**Comment by GitAntoinee** [comment]
you're right janm it wasn't related to the title you finally saw it after almost 1 year xd. I've forgotten this pr

**Comment by ThisTestUser** [comment]
There's too much conflicts in this PR to merge (plus there's already Github Actions).

**Pull request closed without merging**